content/docs/how-to-guides/import-an-existing-database.md

```diff
@@ -4,28 +4,68 @@ redirectFrom:
+
+The command for importing a database from PostgreSQL to Neon appears similar to the following:
+
+```bash
+pg_dump postgres://mypguser:a1B2c3D4e5F6@<hostname>:5432/mydb | psql postgres://myneonuser:a1B2c3D4e5F6@lively-voice-123456.cloud.neon.tech:5432/main
+```
+
+The command for importing a database from one Neon project to another uses two Neon connection strings:
+
+```bash
+pg_dump postgres://myneonuser:a1B2c3D4e5F6@lively-voice-123456.cloud.neon.tech:5432/main | psql postgres://myneonuser:a1B2c3D4e5F6@floral-king-123456.cloud.neon.tech:5432/main
+```
 
-Because `pg_dump` dumps a single database, it does not include information about roles that are stored in the global `pg_authid` catalog. Also, Neon does not support creating users or roles using `psql`. Those can only be created using the Neon Console. If you do not create roles in Neon before importing a database that has roles, you will receive "role does not exist" errors during the import operation. You can ignore this warning. It does not prevent data from being imported.
+If you have multiple databases to import, each database must be imported separately.
 
-You can also configure logical replication to allow Neon to receive a stream of updates from an existing system.
+## Data import notes
 
-Some PostgreSQL features that require access to the local file system are not supported by Neon. For example, tablespaces and large objects are not supported. Please take this into account when importing an existing database to Neon.
+When importing a database, be aware of the following:
 
-In addition to databases, Neon supports importing individual tables. You can do this using the `COPY` command, as you would with a standalone PostgreSQL installation. The only requirement is that the data is transferred through a replication stream, which may affect the performance of other queries, including those unrelated to the table you are copying.
+- If you are importing a database from an archive using `pg_dump` that is  not in plain-text format, use the `pg_restore` utility instead of `psql` to restore the database to Neon.
+- Neon is not able to create databases, so you can not use `pg_dumpall` or `pg_dump` with the `-C` option.
+- Because `pg_dump` dumps a single database, it does not include information about roles stored in the global `pg_authid` catalog. Also, Neon does not support creating roles using `psql`. You can only create roles (users) using the Neon Console. If you do not create roles in Neon before importing a database that has roles, you will receive "role does not exist" errors during the import operation. You can ignore this warning. It does not prevent data from being imported.
+- When importing from a standalone PostgreSQL instance, you can configure logical replication to allow Neon to receive a stream of updates from the PostgreSQL instance.
```

Comment:
Not yet, unfortunately:

```
main=> CREATE SUBSCRIPTION mysub
         CONNECTION 'host=1.2.3.4 user=foo dbname=foodb'
        PUBLICATION mypub;
ERROR:  must be superuser to create subscriptions
```

we should support it, but it will require some amount of work.

Reply:
Item removed.